integration/tests/unowned_cluster/unowned_cluster_test.go

```diff
@@ -226,6 +226,7 @@ var _ = Describe("(Integration) [non-eksctl cluster & nodegroup support]", func(
 				"cluster",
 				"--name", params.ClusterName,
 				"--version", "1.19",
+				"--timeout", "1h30m0s",
```

Comment:
`0s` can be omitted:

```suggestion
				"--timeout", "1h30m",
```